packages/client/src/FC/Base/FC__Globals.re

```diff
@@ -1,36 +1,24 @@
-let commentStyle =
-  Css.[
-    padding3(~top=`em(0.4), ~bottom=`em(0.3), ~h=`em(1.1)),
-    background(FC__Colors.lightGrayBackground),
-    borderRadius(FC__Colors.BorderRadius.tight),
-    color(FC__Colors.Text.LightBackground.p),
-  ];
-
 let load = () => {
   Css.(
     global(
       "body",
       [
-        fontFamily(FC__Colors.Text.standardFont),
+        fontFamily(FC__Settings.Text.standardFont),
         margin(`zero),
         height(`percent(100.0)),
         background(`hex("F0F1F3")),
         fontSize(`px(16)),
-        selector("a", [color(`hex("1256ab"))]),
-        selector("blockquote", commentStyle),
-        selector(
-          "hr",
-          [
-            marginTop(`em(1.)),
-            marginBottom(`em(1.)),
-            borderLeft(`px(0), `none, `hex("000")),
-            borderRight(`px(0), `solid, `hex("000")),
-            borderBottom(`px(0), `none, `hex("000")),
-            borderTop(`px(1), `solid, `hex("f0f2f5")),
-          ],
-        ),
+        lineHeight(`em(1.5)),
+      ],
+    ),
+    global(
+      "button",
+      [
+        fontFamily(FC__Settings.Text.standardFont),
+        fontSize(`px(16)),
+        lineHeight(`em(1.5)),
```

Comment:
Why are these being set here? Can we set them in in the button css instead? Having global properties is a bit of an anti-pattern, unless really needed.

Reply:
The main client of the app has global settings which are imported with Antd lib, so this is why it had been placed here. Will change, thanks.